Examples/hierarchy-tool.lisp

```diff
@@ -17,10 +17,19 @@
                                          (make-rectangle* (+ x1 10) (+ y1 10)
                                                           (- x2 10) (- y2 10))))
       (draw-rectangle* pane x1 y1 x2 y2 :filled nil)
-      (draw-circle* pane (/ (- x2 x1) 2) (/ (- y2 y1) 2) 15
+      (let ((cx (/ (- x2 x1) 2))
```

Comment:
It would be cool (and useful) to show which sheet currently has the input focus.

Reply:
done (with few other improvements)

n.b I've added enter/exit event balance counter and it clearly indicates we have a bug. note that this is not a regression, though I'll try to fix it.